#!/usr/bin/env python
PACKAGE = "orbus_interface"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Slope_time",  double_t, 0, "[s] Time to put velocity motor to zero",                   1.0, 0)
gen.add("Bridge_off",  double_t, 0, "[s] Time to disable bridge, after the speed reaches zero", 2.0, 0)
gen.add("Timeout",     int_t,    0, "[ms] Timeout to start emergency stop of the motors",       100, 100, 5000)

gen.add("restore_defaults", bool_t  , 0, "Restore to the original configuration", False      )

exit(gen.generate(PACKAGE, "hardware_unav", "UnavEmergency"))
